cgi-bin/DW/Controller/OpenID.pm

```diff
@@ -351,7 +351,9 @@ sub openid_login_handler {
         return $r->redirect($check_url);
     }
 
-    my $get_args = $r->get_args;
+    # get_args method returns a Hash::MultiValue,
+    # but LJ::OpenID::consumer needs a plain vanilla hash
+    my $get_args = { %{ $r->get_args } };
```

Comment:
Oh, I feel like I ran into this in the docs at some point. Do we want...

```suggestion
    my $get_args = $r->get_args->mixed;
```

...instead?

Reply:
I had not seen that usage, thanks! I'll update. (Was going to update anyway to drop the warnings commit.)